tests/test_odl2.py

```diff
@@ -0,0 +1,236 @@
+                moby_dick_license_pool.delivery_mechanisms,
+                MediaTypes.AUDIOBOOK_MANIFEST_MEDIA_TYPE,
+                DeliveryMechanism.ADOBE_DRM,
+            )
+        )
+        assert moby_dick_audio_book_adobe_drm_delivery_mechanism is not None
+
+        moby_dick_audio_book_lcp_drm_delivery_mechanism = (
+            self._get_delivery_mechanism_by_drm_scheme_and_content_type(
+                moby_dick_license_pool.delivery_mechanisms,
+                MediaTypes.AUDIOBOOK_MANIFEST_MEDIA_TYPE,
+                DeliveryMechanism.LCP_DRM,
+            )
+        )
+        assert moby_dick_audio_book_lcp_drm_delivery_mechanism is not None
+
+        moby_dick_audio_book_feedbooks_drm_delivery_mechanism = (
+            self._get_delivery_mechanism_by_drm_scheme_and_content_type(
+                moby_dick_license_pool.delivery_mechanisms,
+                MediaTypes.AUDIOBOOK_MANIFEST_MEDIA_TYPE,
+                DeliveryMechanism.FEEDBOOKS_AUDIOBOOK_DRM,
+            )
+        )
+        assert moby_dick_audio_book_feedbooks_drm_delivery_mechanism is not None
+
+        assert 1 == len(moby_dick_license_pool.licenses)
+        [moby_dick_license] = moby_dick_license_pool.licenses
+        assert (
+            "urn:uuid:f7847120-fc6f-11e3-8158-56847afe9799"
+            == moby_dick_license.identifier
+        )
+        assert (
+            "http://www.example.com/get{?id,checkout_id,expires,patron_id,passphrase,hint,hint_url,notification_url}"
+            == moby_dick_license.checkout_url
+        )
+        assert "http://www.example.com/status/294024" == moby_dick_license.status_url
+        assert datetime.datetime(2016, 4, 25, 10, 25, 21) == moby_dick_license.expires
+        assert 10 == moby_dick_license.remaining_checkouts
+        assert 10 == moby_dick_license.concurrent_checkouts
+
+        # 3. Make sure that work objects contain all the required metadata
+        assert isinstance(works, list)
+        assert 1 == len(works)
+
+        moby_dick_work = self._get_work_by_identifier(
+            works, "urn:isbn:978-3-16-148410-0"
+        )
+        assert isinstance(moby_dick_work, Work)
+        assert moby_dick_edition == moby_dick_work.presentation_edition
+        assert 1 == len(moby_dick_work.license_pools)
+        assert moby_dick_license_pool == moby_dick_work.license_pools[0]
+
+        # 4. Make sure that the failure is covered
+        assert 1 == len(failures)
+        moby_dick_failures = failures["9781234567897"]
+
+        assert 1 == len(moby_dick_failures)
+        [moby_dick_failure] = moby_dick_failures
+        assert isinstance(moby_dick_failure, CoverageFailure)
+        assert "9781234567897" == moby_dick_failure.obj.identifier
+        assert (
+            u"ODL publication 'urn:isbn:9781234567897' does not contain neither 'licenses' subcollection "
+            u"nor an Open-Access Acquisition Link (http://opds-spec.org/acquisition/open-access)"
```

Comment:
The language here should probably be "...contains neither ... nor ..." or "...does not contain either ... or ...":
```
ODL publication 'urn:isbn:9781234567897' contains neither a 'licenses' subcollection nor an Open-Access Acquisition Link...
```
or
```
ODL publication 'urn:isbn:9781234567897' does not contain either a 'licenses' subcollection or an Open-Access Acquisition Link...
```

but this can be updated in another PR in coordination with the corresponding change to the [Web Publication Manifest Parser](https://github.com/ThePalaceProject/webpub-manifest-parser/blob/3d186af50aadd89c6be008530b365ef46c6b3185/src/webpub_manifest_parser/odl/semantic.py#L113-L117), where the message originates.

Reply:
I created [PR # 8](https://github.com/ThePalaceProject/webpub-manifest-parser/pull/8) in webpub-manifest-parser repo.

Reply:
Sounds good.